android/res/layout/list_projects.xml

```diff
@@ -2,7 +2,7 @@
 <LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
               android:layout_width="match_parent" android:layout_height="match_parent">
 
-  <ListView android:id="@android:id/list" android:layout_width="match_parent"
+  <ListView android:id="@+id/list" android:layout_width="match_parent"
```

Comment:
just out of curiosity, what is the +?

Reply:
+id tells android to add a new id in the resources so you can call it from java with `R.`